makefile.vc

```diff
@@ -62,14 +62,15 @@ test: $(app) venv\.test.installed
 
 ci-test:
 	set CI=true
-	pytest \
-		--verbose \
-		-p no:sugar \
-		--cov-report term \
-		--cov-report html:test-results\coverage/ \
-		--junit-xml=test-results\junit.xml \
-		--benchmark-enable \
-		-p no:xdist
+	FOR /L %%I IN (0, 1, 9) DO \
+		pytest \
+			--num-shards 10 \
```

Comment:
are the shards deterministic?

Reply:
I think yes - 
certainly if you run the exact same command twice in a row without changing anything, the tests assigned to a particular shard is the same each time. But I think there is a stronger guarantee: a test named X will always be in shard `hash(X) % num-shards` - so as long as the test keeps the same name and you keep the same number of shards, it will always be in the same shard.
This guess about how it works is based on a single line of documentation "Shards tests based on a hash of their test name", and the fact that I observe different shards have different numbers of tests in them.

Reply:
If it's using `hash()` then I think it's only guaranteed to be deterministic in the same interpreter instance? Though `PYTHONHASHSEED` affects for scalar objects, hashes of lists/dicts/objects are derived from their memory addresses.

Something based on a hashlib algorithm would be more useful if it's possible, since it'd be repeatable across CI runs.

@craigds Didn't we go through this with dandelion/something?

Reply:
... well, the name of a test won't be a list/dict/object... just a string. Furthermore the rest of that line of documentation says "Shards tests based on a hash of their test name enabling easy parallelism across machines" - so I think the hash function (whatever it is) is supposed to be relatively machine-and-context independent, although I wouldn't know if it will change if we, for instance, upgrade python. 

I kind of thought you would be happy the windows test is finally green... it seemed preferable to me. Let me know if I should roll it back

Reply:
is all good :)